src/intl/en/glossary.json

```diff
@@ -245,6 +245,8 @@
   "pos-definition": "A method by which a cryptocurrency blockchain protocol aims to achieve distributed <a href=\"/glossary/#consensus\">consensus</a>. PoS asks users to prove ownership of a certain amount of cryptocurrency (their \"stake\" in the network) in order to be able to participate in the validation of transactions. <a href=\"/developers/docs/consensus-mechanisms/pos/\">More on proof-of-stake</a>.",
   "pow-term": "Proof-of-work (PoW)",
   "pow-definition": "A piece of data (the proof) that requires significant computation to find. <a href=\"/developers/docs/consensus-mechanisms/pow/\">More on proof-of-work</a>.",
+  "proto-danksharding-term": "Proto-Danksharding",
+  "proto-danksharding-definition": "A new transaction type which accepts \"blobs\" of data for Ethereum. This \"blob\" data is temporarily stored on the beacon chain for 4096 epochs (~18.2 days), and can optionally be pruned after to help reduce hardware requirements for node operators.",
```

Comment:
the definition here is not the same as the one provided in https://github.com/ethereum/ethereum-org-website/issues/9026

Reply:
Yeah, I believe this definition builds on what was suggested, but the iteration is an improvement.